.github/workflows/deploy-cs-dashboards.yaml

```diff
@@ -5,8 +5,6 @@ on:
     types: [closed]
     branches:
       - "main"
-    paths:
-      - terraform/dasboards/Near/chain-signaures
```

Comment:
Not sure I understand what this change mean

Reply:
I think this may have been breaking the automated workflow run, I was hoping it was a working directory flag, but I don't think it works like I thought.

Reply:
we can test it of you want